app/views/users/show_orders.html.erb

```diff
@@ -0,0 +1,22 @@
+<% counter = 0 %>
+<% @user.orders.each do |order| %>
+  <h2>ORDER#<%= counter = counter+1 %></h2>
+  customer: <%= order.name %>
+  <br>
+  email: <%= order.email %>
+  <br>
+  <h3>line items:</h3>
+  <br>
+  <% total_price = 0 %>
+  <% order.line_items.each do |line_item| %>
+    <%= Product.find(line_item.product_id).title %>
```

Comment:
not the right way. Please use association here.

Reply:
add eager loading as well